prisma/migrations/20240202103321_added_feedback_table/migration.sql

```diff
@@ -0,0 +1,12 @@
+-- CreateTable
+CREATE TABLE "feedback" (
+    "id" UUID NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "name" VARCHAR(128) NOT NULL,
+    "phone" VARCHAR(13) NOT NULL,
+    "email" VARCHAR(254),
+    "callAgain" BOOLEAN NOT NULL,
+    "message" VARCHAR(320) NOT NULL,
+
+    CONSTRAINT "feedback_pkey" PRIMARY KEY ("id")
```

Comment:
🟡 i think there was request to be able to check feedback as reviewed and ability for admins to add notes to feedback, its two more fields

Reply:
I will add notes field, but there is no strict requirements about checking as reviewed. Ticket: https://trello.com/c/rZRMkeGN/13-%D0%B0%D0%BF-%D0%BC%D0%B5%D0%BD%D0%B5%D0%B4%D0%B6%D0%BC%D0%B5%D0%BD%D1%82-%D0%B7%D0%B2%D0%BE%D1%80%D0%BE%D1%82%D0%BD%D1%8C%D0%BE%D0%B3%D0%BE-%D0%B7%D0%B2%D1%8F%D0%B7%D0%BA%D1%83).  This functionality should be discussed later as written in ticket